extra/release_tool.py

```diff
@@ -1103,7 +1103,13 @@ def check_tag_availability(state):
 
 def repo_sort_key(repo):
     """Used in sorted() calls to sort by Git name."""
-    return repo.name
+    if repo.name.endswith("-enterprise"):
```

Comment:
maybe we should sync feature-monitor with master, instead? (same with the other commit)